src/components/help/help.component.js

```diff
@@ -14,7 +14,7 @@ const Help = (props) => {
     href,
     helpId,
     children,
-    tabIndex,
+    tabIndexOverride,
```

Comment:
This should remain as `tabIndex` instead of `tabIndexOverride` I think this is because of the rebase. This would have been `tabIndexOverride` then changed to `tabIndex` in another branch (and now in master) but because of the rebase, it's changing it back to `tabIndexOverride`... I think.

Let's keep it as the standard `tabIndex` might need to check for other references to `tabIndexOverride`

Reply:
https://github.com/Sage/carbon/commit/cc097a56940f85be11a5ac6c9f9d4e147fca207c

Reply:
reverted